spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -8,12 +8,18 @@
 Tests for explorer.py
 """
 
+# Standard imports
+import os
+import os.path as osp
+
 # Test library imports
```

Comment:
Change to "Third party imports"

Reply:
The highlighted line here (its always the *last* line in Github reviews, not the first, FYI) is ``# Test library imports``, not ``# Standard library imports``.

Reply:
Looks great, thanks!